src/sentry/static/sentry/app/views/eventsV2/results.tsx

```diff
@@ -298,10 +366,29 @@ class Results extends React.Component<Props, State> {
                   setError={this.setError}
                   onChangeShowTags={this.handleChangeShowTags}
                   showTags={showTags}
+                  confirmedQuery={confirmedQuery}
                 />
               </StyledMain>
               {showTags ? this.renderTagsTable() : null}
             </ContentBox>
+            <Confirm
+              priority="primary"
+              header={<strong>May lead to thumb twiddling</strong>}
```

Comment:
```suggestion
              header={<strong>{t('May lead to thumb twiddling')}</strong>}
```

Reply:
:+1: updated, also added it to `confirmText` and `cancelText`